publish.sh

```diff
@@ -13,6 +14,26 @@ TARGET_KUBEVIRT_REPO="quay.io/kubevirt"
 TARGET_GIT_REMOTE="https://kubevirt-bot@github.com/kubevirt/kubevirtci.git"
 export CRI_BIN=${CRI_BIN:-$(detect_cri)}
 
+IMAGES_TO_BUILD=()
+IMAGES_TO_RETAG=()
+
+function run_provision_manager() {
+  # TODO need to install jq
+  json_result=$(${CRI_BIN} run --rm -v $(pwd):/workdir:Z quay.io/kubevirtci/gocli pman)
+  echo "INFO: Provision manager results: $json_result"
+
+  while IFS=":" read -r key value; do
+      if [[ "$value" == "true" ]]; then
+          IMAGES_TO_BUILD+=("$key")
+      else
+          IMAGES_TO_RETAG+=("$key")
+      fi
+  done < <(echo "$json_result" | jq -r 'to_entries[] | "\(.key):\(.value)"')
```

Comment:
@brianmcarey 

Is it fine that we install please jq on the bootstrap image?
I will create the PR if so

Reply:
It should be there already - https://github.com/kubevirt/project-infra/blob/43ae34e031803dc4d47fba5518e3411e9e514fc6/images/bootstrap/Dockerfile#L41

Reply:
Right just saw thanks